script.js

```diff
@@ -0,0 +1,167 @@
+const countriesGrid = document.querySelector('.countries-grid');
+let countryFetchedData = null;
+
+async function fetchCountriesData() {
+    if (countryFetchedData) {
+        return;
+    }
+    try {
+        const response = await fetch('./CountriesData.json');
+        if (!response.ok) {
```

Comment:
Also you can save your strings inside constants file or variable and not use them as hardcoded values